include/caliper/Annotation.h

```diff
@@ -134,6 +134,16 @@ class Annotation
     /// \param opt  %Attribute flags. Bitwise OR combination 
     ///   of \ref cali_attr_properties values.
     Annotation(const char* name, int opt = 0);
+    
+    using MetadataListType = std::map<const char*, Variant>;
```

Comment:
I think `using` is C++11 ... would need `typedef` for C++98. Or just hard-code it as std::map<...> without an alias.

Reply:
Fixed with typedef. Good catch